docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8081:8081"
+    environment:
+      - ENV=local
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: mysql:1.0 
```

Comment:
The MySQL image tag `1.0` is not a valid tag for the official MySQL image. Consider using a valid tag such as `latest` or a specific version number like `5.7` or `8.0`.